packages/react-ui-core/src/Carousel/Carousel.js

```diff
@@ -212,11 +215,12 @@ export default class Carousel extends Component {
           items={this.items}
           renderItem={this.renderItem}
           onSlide={this.onSlide}
-          showThumbnails={false}
+          showThumbnails={showThumbs}
```

Comment:
Why not name the prop `showThumbs`? Seems like having two different names just confuses things.

Reply:
fair enough--I think that makes sense.